sdk/openai/Azure.OpenAI.Inference/api/Azure.OpenAI.Inference.netstandard2.0.cs

```diff
@@ -0,0 +1,98 @@
+namespace Azure.OpenAI.Inference
```

Comment:
I'm assuming the package name and namespace capitalization was approved by the archboard, since it goes against the typical C# convention of only capitalizing the first letter of ackronyms.

Reply:
We will decide / change that at the archboard review